test/system/reports_test.rb

```diff
@@ -648,6 +648,23 @@ def wait_for_watch_change
     assert_selector('.card-list-item__user')
   end
 
+  test 'user role class is displayed correctly in reports' do
+    visit_with_auth reports_path, 'kimura'
+
+    within('.card-list-item__user', match: :first) do
+      assert_selector('
+        span.a-user-role.is-student,
+        span.a-user-role.is-admin,
+        span.a-user-role.is-retired,
+        span.a-user-role.is-hibernationed,
+        span.a-user-role.is-mentor,
+        span.a-user-role.is-adviser,
+        span.a-user-role.is-graduate,
+        span.a-user-role.is-trainee
+        ')
```

Comment:
```suggestion
      assert_selector('
        span.a-user-role.is-student,
        span.a-user-role.is-admin,
        span.a-user-role.is-retired,
        span.a-user-role.is-hibernationed,
        span.a-user-role.is-mentor,
        span.a-user-role.is-adviser,
        span.a-user-role.is-graduate,
        span.a-user-role.is-trainee
      ')
```

こっちらのインデントの方が一般的かもです〜